app/src/main/java/com/example/dogbook/main/viewholders/PostViewHolder.java

```diff
@@ -52,6 +59,10 @@ public void bind(Post post) {
             ivPostPicture.setVisibility(View.VISIBLE);
             Glide.with(context).load(postPicture.getUrl()).into(ivPostPicture);
         }
+
+        //tvLikeCount.setText(post.getNoLikes());
+        //tvCommentCount.setText(post.getNoComments());
+        //cbLike.setChecked(post.isLiked());
```

Comment:
Let's not commit this, even though it is skeleton code -- we can add this once you've setup your query